src/config/.hound.yml

```diff
@@ -0,0 +1,2 @@
+eslint:
```

Comment:
make research and find out whether adding extra fields within this eslint such as the ones below is necessary or recommended:
```
config_file: .eslintrc.json
  version: x.x.x
  extensions:
    - .ts
```

also adding other fields like:
`additional_dependencies:`

Reply:
Yes, I have worked on it and I have configured HoundCi to work with the eslint.config.mjs